tests/integration_tests/functional/test_mmds.py

```diff
@@ -13,6 +17,21 @@ def _assert_out(stdout, stderr, expected):
     assert stdout.read() == expected
 
 
+def _metadata_json(test_microvm, metadata_file):
+    """Configure a microvm with metadata from the command line."""
+    metadata_path = os.path.join(
+        test_microvm.path,
+        os.path.basename(metadata_file)
+    )
+    with open(metadata_file) as f1:
```

Comment:
Is this block simply copying a file to a path? We can do that in one line using `shutil.copy()`.